code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm

```diff
@@ -76,7 +76,7 @@
 	if(iscultist(L))
 		to_chat(L, "<span class='heavy_brass'>\"Watch your step, wretch.\"</span>")
 		L.adjustBruteLoss(10)
-		L.Knockdown(80, FALSE)
+		L.Knockdown(120, FALSE)
```

Comment:
I don't think you've buffed this how you think you've buffed this.

Reply:
8 seconds to 12 seconds

Reply:
ahh you're right